USF-2012: Create Terms and Conditions documentation (#204)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/checkout/functions.mdx

fetchCartData();
```

## getCheckoutAgreements

The `getCheckoutAgreements` function fetches the list of terms and conditions configured in the Admin Panel using the [`checkoutAgreements`](https://developer.adobe.com/commerce/webapi/graphql/schema/checkout/queries/agreements/) query.

```ts
export const getCheckoutAgreements = async (): Promise<CheckoutAgreement[]>
```

### Returns

> The function returns a promise that resolves to the array of transformed checkout agreements data fetched from the API or an empty array if no agreements enabled found.

Reviewer: ```suggestion
The function returns a promise that resolves to an array of checkout agreements or an empty array if none are enabled.
```
